optimum/intel/generation/modeling.py

```diff
@@ -81,9 +81,34 @@ def load_model(file_name: Union[str, Path]):
         torch.jit.freeze(model.eval())
         return model
 
+    @staticmethod
+    def jit_trace(model: PreTrainedModel, task: str, config: PretrainedConfig, use_cache: bool = True):

```

Comment:
I think it would make sense to create a separate function `trace` (working for all architectures not only causal LM) and to use it in `TSModelForCausalLM._from_transformers` instead of having `jit_trace` and `export_model` methods.